sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,147 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    """Coulomb kinetic friction with Stribeck and viscous effect.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+        F = f_c * sign(v) + (f_max - f_c) * exp(-(v / v_s)**2) + sigma * v
+
+    where `f_c` is the Coulomb friction constant, `f_max` is the maximum static friction force,
+    `v_s` is the coefficient of sliding friction, `sigma` is the viscous friction constant,
+    and `v` is the relative velocity.
+
+    The default friction force is `F = mu_k * N`, where N is the normal force.
+    When specified, the actuator includes:
+    - Stribeck effect: `(mu_s - mu_k) * N * exp(-(v / v_s)**2)`
+    - Viscous effect: `sigma * v`
+
+    In case we include all effects, the full actuator represents:
+    `F = mu_k * N * sign(v) + (mu_s - mu_k) * N * exp(-(v / v_s)**2) + sigma * v`.
+
+    Please note that this actuator assumes slip and applies a force opposite to the velocity direction.
+
+    Parameters
+    ==========
+
+    normal_force : Expr
+        The normal force between the surfaces.
+    mu_k : Expr
+        The coefficient of kinetic friction.
+    pathway : PathwayBase
+        The pathway that the actuator follows.
+    v_s : Expr
+        The coefficient of sliding friction (COF).
+    viscous_coefficient : Expr
+        The viscous friction coefficient.
+    mu_s : Expr, optional
+        The coefficient of static friction. Defaults to `mu_k`, meaning the Stribeck effect evaluates to 0 by default.
+
+    References
+    ==========
+
+    Learn Multibody Dynamics. Available at:
+      https://moorepants.github.io/learn-multibody-dynamics/loads.html#friction.
+    Chalmers University of Technology, Open Digital Repository. Available at:
+      https://odr.chalmers.se/server/api/core/bitstreams/6316ee50-2ffc-4c29-97c1-c5e7a2737a5f/content.
```

Comment:
Could you format the references as described [here](https://docs.sympy.org/dev/contributing/docstring.html#references-section)?

Reply:
I reformatted the reference as
```
References
==========

.. [Moore2022] https://moorepants.github.io/learn-multibody-dynamics/loads.html#friction.
.. [Flores2023] Paulo Flores, Jorge Ambrosio, Hamid M. Lankarani,
        "Contact-impact events with friction in multibody dynamics: Back to basics",
        Mechanism and Machine Theory, vol. 184, 2023. https://doi.org/10.1016/j.mechmachtheory.2023.105305.
.. [Rogner2017] I. Rogner, "Friction modelling for robotic applications with planar motion",
        Chalmers University of Technology, Department of Electrical Engineering, 2017.
```